### ISSUE TITLE
Shapely with py2app -> crash when running the app due to geos

### ISSUE THREAD

**Issue by AGMoreno** [Toblerity/Shapely#292]
Hi,

Working in Mac OS X Yosemite with python 3.4

My file, test.py, runs a code with Shapely. When using the alias mode of py2app (python test.py py2app -A) the application runs perfectly. Now, I create the final standalone (python test.py py2app) and the output app crashes when trying to run it. The reason:

14/07/15 16:11:47,603 test[9027]: OSError: dlopen(/Users/myHouse/Desktop/me/test/dist/test.app/Contents/Resources/lib/python3.4/shapely/.dylibs/libgeos_c.1.dylib, 6): Symbol not found: __ZN4geos2io9WKBReader4readERSi

If I do not include Shapely under the "packages" option in the setup.py file that py2app is reading, the app runs perfectly. That is because (I guess) that in this later scenario the app uses the shapely copy in the computer (it does not run in other computers because of libgeos). 

So basically I do not know how to use Shapely with the geometry module in py2app. Ideally, I just need to specify the package name in the setup.py file but then geos crashes. Maybe locations? It looks like the app cannot find things in their proper place.

Any help really appreciated!

Cheers,
A!

**Comment by sgillies** [comment]
@AGMoreno sorry, I don't have the time to look into using Shapely with py2app, but if you have a patch with tests that helps you I'd be happy to review it.
